src/main.js

```diff
@@ -101,9 +103,15 @@ function toggleAllUsers(status = false) {
     onToggleUser();
 }
 
-function renderResults(resultsList) {
+function refreshPagination() {
+    const paginationCount = document.getElementById('current-page');
+    paginationCount.innerHTML = `${currentPage}/${getMaxPage()}`;
+}
+
+function renderResults() {
+    refreshPagination();
     // Shallow-copy to avoid altering original.
-    const sortedList = [...resultsList].sort((a, b) => (a.username > b.username ? 1 : -1));
+    const sortedList = getCurrentPageUnfollowers();
```

Comment:
Actually the `sortedList` variable was only used due to the sorting.
Now that it's done in the function, you could just do this below ->

```ts
getCurrentPageUnfollowers().forEach(user => {
```

and delete this line entirely.
i.e. - no reason to keep it in a variable if it's short and clean enough to inline.

Reply:
Also, should probably either remove the comment above this line, or move it to the `getCurrentPageUnfollowers()`.
Since that's where it's actually being cloned now.